tests/js/jest.config.js

```diff
@@ -10,9 +10,7 @@ module.exports = {
 	coveragePathIgnorePatterns: [
 		'<rootDir>/build/',
 		'<rootDir>/node_modules/',
-		'assets/js/googlesitekit-api.js',
-		'assets/js/googlesitekit-data.js',
-		'assets/js/googlesitekit-datastore-site.js',
+		'assets/js/googlesitekit-*.js',
```

Comment:
Oddly this isn't ignoring the files properly. We can sort it out later, but I see:

<img width="1094" alt="Screenshot 2020-03-04 at 22 34 02" src="https://user-images.githubusercontent.com/90871/75929334-44141d00-5e68-11ea-9a84-20ad7dc8093b.png">

when the wildcard is used. Maybe it needs to be a regex. I'll try that and commit if so; otherwise we can deal with it later 😅

Reply:
Update: regex works; added in 22315be0465da48e88f51093d671aa268712e52b